SocketRocket/SocketRocket/NSRunLoop+ARTSRWebSocket.m

```diff
@@ -24,4 +24,11 @@ + (NSRunLoop *)ARTSR_networkRunLoop
     return [ARTSRRunLoopThread sharedThread].runLoop;
 }
 
++ (void)ARTSR_waitForNetworkRunLoop
+{
+    while ([ARTSRRunLoopThread sharedThread].runLoop == nil) {
```

Comment:
I thought about it first, but then decided to go the simplest way possible just to avoid too much changes in SR library, which is going, as I've already mentioned in the comment above, affect very few people due to it being replaced with the `NSURLSession` web socket in another PR.
Also, `ARTSR_waitForNetworkRunLoop` method exits almost immediately, since the time between `start()` and `main()` can be negligible.
WDYT @QuintinWillison

Reply:
Resolving this one since @QuintinWillison mentioned on Slack he'd prefer to focus on getting this released.